### ISSUE TITLE
LDAP-342: Support Spring Data REST

### ISSUE THREAD

**Issue by spring-projects-issues** [spring-projects/spring-ldap#374]
[Rob Winch](https://jira.spring.io/secure/ViewProfile.jspa?name=rwinch) (Migrated from [LDAP-342](https://jira.spring.io/browse/LDAP-342?redirect=false)) said:

It would be nice to be able to export Spring LDAP repositories as REST endpoints using Spring Data REST. There is an example of this listed on LDAP-341

**Comment by spring-issuemaster** [comment]
This issue relates to #373

**Comment by rfkarlin** [comment]
Is there any update on this?  I was hoping to be able to use this as well.

**Comment by rwinch** [comment]
@rfkarlin Thank you for your interest. At the moment there is no updates on this (mostly due to lack of time). 

If it is something that interests you, we would love to see a PR :)

**Comment by rfkarlin** [comment]
Hi Rob,

Do you mean that you would like me to work on the fix?

—Robin
Robin F Karlin
Principal Systems Software Engineer
Identity Services
Carnegie Mellon University
5000 Forbes Avenue
Pittsburgh, PA 15213-3890
Office: 412-268-9046




On Mar 9, 2016, at 12:17 PM, Rob Winch <notifications@github.com<mailto:notifications@github.com>> wrote:


@rfkarlin<https://github.com/rfkarlin> Thank you for your interest. At the moment there is no updates on this (mostly due to lack of time).

If it is something that interests you, we would love to see a PR :)

**Comment by rwinch** [comment]
@rfkarlin

> Do you mean that you would like me to work on the fix?

Yes. I'd love for you to send a [Pull Request (PR)](https://help.github.com/articles/using-pull-requests/)
